app/serializers/merchant_revenue_serializer.rb

```diff
@@ -0,0 +1,19 @@
+class MerchantRevenueSerializer
+
+  def initialize(id, revenue)
+    @id = id
+    @revenue = revenue
+  end
+
+  def serialize
+    {
+      data: {
+        id: @id,
+        type: "merchant_revenue",
+        attributes: {
+          revenue: @revenue
+        }
+      }
+    }
+  end
```

Comment:
decided to go with custom JSON response for practice, AND had difficulty with `jsonapi-serializer` formatting to include a Merchant id.